config/brakeman.ignore

```diff
@@ -0,0 +1,1417 @@
+      "code": "(Unresolved Model).new.description_html",
+      "render_path": [
+        {
+          "type": "controller",
+          "class": "Admin::AmbassadorTasksController",
+          "method": "index",
+          "line": 6,
+          "file": "app/controllers/admin/ambassador_tasks_controller.rb",
+          "rendered": {
+            "name": "admin/ambassador_tasks/index",
+            "file": "app/views/admin/ambassador_tasks/index.html.haml"
+          }
+        }
+      ],
+      "location": {
+        "type": "template",
+        "template": "admin/ambassador_tasks/index"
+      },
+      "user_input": null,
+      "confidence": "High",
+      "cwe_id": [
+        79
+      ],
+      "note": "@sethherr: Mark them all (Cross-Site Scripting vulnerabilities in ambassador_tasks/index) as not concerns"
+    },
+    {
+      "warning_type": "Cross-Site Scripting",
+      "warning_code": 2,
+      "fingerprint": "0ffefb46fb11b12f2129c2b96c51cade952dc489a83a3a4886062071e2ad6410",
+      "check_name": "CrossSiteScripting",
+      "message": "Unescaped model attribute",
+      "file": "app/views/stolen_bike_listings/index.html.haml",
+      "line": 21,
+      "link": "https://brakemanscanner.org/docs/warning_types/cross_site_scripting",
+      "code": "Blog.friendly_find(324).body",
+      "render_path": [
+        {
+          "type": "controller",
+          "class": "StolenBikeListingsController",
+          "method": "index",
+          "line": 18,
+          "file": "app/controllers/stolen_bike_listings_controller.rb",
+          "rendered": {
+            "name": "stolen_bike_listings/index",
+            "file": "app/views/stolen_bike_listings/index.html.haml"
+          }
+        }
+      ],
+      "location": {
+        "type": "template",
+        "template": "stolen_bike_listings/index"
+      },
+      "user_input": null,
+      "confidence": "Medium",
+      "cwe_id": [
+        79
+      ],
+      "note": ""
+    },
+    {
+      "warning_type": "Dangerous Send",
+      "warning_code": 23,
+      "fingerprint": "13270a2b6e6d5bc6e590beca380143b628da6f0b3352e4dfd0a8901de4634e8c",
+      "check_name": "Send",
+      "message": "User controlled method execution",
+      "file": "app/controllers/organized/impound_claims_controller.rb",
+      "line": 74,
+      "link": "https://brakemanscanner.org/docs/warning_types/dangerous_send/",
+      "code": "impound_claims.public_send((params[:search_status] or available_statuses.first))",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "Organized::ImpoundClaimsController",
+        "method": "available_impound_claims"
+      },
+      "user_input": "params[:search_status]",
+      "confidence": "High",
+      "cwe_id": [
+        77
+      ],
+      "note": "@sethherr: All these sends are for whitelisted methods (from https://github.com/bikeindex/bike_index/pull/2560)"
```

Comment:
Small nitpick here. Could you change this (and all other mentions) to "allowlisted"? Thanks!